### ISSUE TITLE
[Bug?]: Preloading font script in Safari breaks font weights

### ISSUE THREAD

**Issue by GianaS** [pluralsight/pando#1068]
### Duplicates

- [X] I have searched the existing issues

### Latest version

- [X] I have tested the latest version

### Current behavior 😯

Following [the docs](https://design.pluralsight.com/docs/development/getting-started/installation/#ps-tt-commons-font) to install the fonts, we see the font correctly loading on our page in all browsers.  However, in Safari, the variable font weights are not working (everything looks like font weight 400).  We found the only way to get the weights working again in Safari is to remove the font preload script.

### Expected behavior 🤔

The variable font weights should work in all browsers when using the font preload script.

### Steps to reproduce 🕹

Steps:

1. Use both documented scripts to preload & load the font.
```
<link
  rel="preload"
  href="https://fonts.pluralsight.com/ps-tt-commons/v1/ps-tt-commons-variable-roman.woff2"
  as="font"
  type="font/woff2"
  crossorigin
/>
<link
  rel="stylesheet"
  href="https://cdn.jsdelivr.net/npm/@pluralsight/design-tokens@next/fonts.css"
/>
```

2. Check the font weights in Safari & another browser.  They will be different.

3.  Remove the preload script.

4.  Check the font weights in Safari and another browser.  They will be the same.

### Context 🔦

We are trying to cutover to Pando.  We are not blocked because we can just remove the preload script.  However as the docs suggest, the preload script provides performance optimizations and we would like to benefit from them.

### Your environment 🌎

```shell
Issue occurs on multiple machines, but I am using Safari versions 16.0 & 16.2 on macOS Monterey version 12.6.
```

**Comment by caseybaggz** [comment]
Possible variable font issue not a preload one. Preload is supported on all browsers, however, the variable font is still experimental. Unfortunately, the PS font is only a variable font.

https://caniuse.com/variable-fonts

**Comment by GianaS** [comment]
Hello!

We are now experiencing issues with the font-weight in other browsers even though we are no longer using the preload script.  This leads us to believe that is not the cause of the issue.  However, we've seen the other issues created for FVS and we're led to believe this could be related to it or potentially something else in the new DS normalize.css script.  We removed the classic DS normalize script and still experience the flickering font-weight, so we know it's not an issue with that.

Below, I added a screen recording to demonstrate the issue.  There are inconsistent font weights on both elements where we explicitly set the font-weight (ie. the content type label on Search result rows) **and** elements we do not, where the weight should be inherited from normalize (ie. the Search placeholder text in the Prism search bar, which is outside of our Bounded Context).

This issue is a blocker for us on releasing the rebranding in Search/Browse, which could affect the release of 1.3 if not resolved.  Is it possible to reopen this issue?

https://user-images.githubusercontent.com/40693732/218788711-4f28c22f-caa7-47de-85ae-3853fc5ea1f1.mov

**Comment by caseybaggz** [comment]
Just copying what was mentioned in a Slack discussion - Safari has a known bug with variable fonts that requires the use of the latest browser combined with the latest OS to work. Likewise, we have no control over the fact that the PS font is a variable font which requires the use of `font-variation-settings` in order to change the weight (i.e. `font-weight` will not work).

Likewise, we cannot guarantee a successful result in our supported browser range if the usage we recommend is altered (i.e. removing the preload script for fonts). The new normalize is the old normalize with a couple of extra new features unrelated to fonts).

So, in a nutshell, when used following the installation guide, the only users that will not see font-weights outside of native HTML browser support will be a small handful of Safari users who are not on the latest browser and do not have the latest OS system which is an acceptable fallback for both the Classic and Pando DS - meaning it should not be a blocker to the team releasing.

**Comment by caseybaggz** [comment]
Update regarding Safari also mentioned in #1091

**Comment by caseybaggz** [comment]
@GianaS if you revert to following the recommended installation usage and it still doesn't work, I'm happy to hop on a call with you to understand the project/setup a little more and see what is different from how we are producing the results successfully in our tests and production environments (client and server apps).

**Comment by msisti** [comment]
Hi @caseybaggz ! Following up on [this thread](https://pluralsight.slack.com/archives/C03D8G0V42G/p1676391981043349) from yesterday, Giana and I have spent a lot of time trying to replicate the issue in a Sandbox environment, as well as narrow down what was causing the issue for us locally. As of yesterday afternoon, we have not been able to enter into the heavier font weight state that we demonstrated in the video in a replicated environment or even in our local environments. However, in our Storybook environment, we somehow still have that heavier font weight state which has fortunately allowed us to compare the computed styles between the “thick” and “thin” states.

In Storybook, we can see the FVS setting a heavier font weight for the selected element (the content type label in the row). This setting (coming from [https://cdn.jsdelivr.net/npm/@pluralsight/design-tokens@next/styles/_typography.scss](https://cdn.jsdelivr.net/npm/@pluralsight/design-tokens@next/styles/_typography.scss)) is not on the element in our local environments (comparison of the two states shown in attached screenshots), and we see that as of [this commit](https://github.com/pluralsight/pando/commit/c98a56c6999151d43f54784da1306231cb334977), those FVS attributes have been removed. This leads us to believe that our Storybook environment must be accessing a cached version of that file that still has the FVS attributes. Now that we are in a steady state locally of the “thinner” font-weight, this must actually be the up-to-date state for us to be in. We must have been in a weird state yesterday with it toggling between accessing the new and previous versions of that stylesheet. We think then that the mystery is solved for what was making the font weights intermittently change yesterday when we had not introduced any recent changes to them in our app! 🎉 

To prevent something like this happening in the future, would y’all recommend we specify a specific version in our stylesheet script links? We imagine this was introduced from using `design-tokens@next` per the [docs](https://design.pluralsight.com/docs/development/getting-started/installation#normalizecss).

<img width="1502" alt="Screen Shot 2023-02-15 at 11 36 58 AM" src="https://user-images.githubusercontent.com/10967659/219096247-bac2c92a-cb66-43d0-9171-4bc55de1232b.png">

<img width="1490" alt="Screen Shot 2023-02-15 at 11 37 25 AM" src="https://user-images.githubusercontent.com/10967659/219096254-0ad30fdd-2799-4686-85f3-4301599d76d7.png">

**Comment by caseybaggz** [comment]
@msisti ah! That's totally a doc bug on our part. The links *should* be using the strict version for whichever docs you are viewing, so `next` should only be used if you are using the `next` build. Great find and glad it worked out!

**Comment by msisti** [comment]
Hey @caseybaggz, thank you! We wanted to verify the path to the stylesheets we should be using. We currently have v0.3.1 installed for design-tokens and headless-styles, so we updated the stylesheet links to be `https://cdn.jsdelivr.net/npm/@pluralsight/design-tokens@0.3.1/fonts.css` and `https://cdn.jsdelivr.net/npm/@pluralsight/design-tokens@0.3.1/npm/normalize/normalize.css`. Are those correct? The styles look off locally, though the stylesheets load in successfully.

<img width="1452" alt="Screen Shot 2023-02-15 at 2 35 37 PM" src="https://user-images.githubusercontent.com/10967659/219134187-c8875be5-878e-4710-bc66-5c30e6ed4443.png">

**Comment by caseybaggz** [comment]
@msisti We actually *just* cut the new release and updated the docs bug that you found, so, the docs site is now reliable for whichever version you would like to use.

But - just to make sure you have a better understanding of the Pando engineering design, 

> All pando packages should match. It doesn't matter what it is - they should all use the same version system in order to work with each other.

There are a bunch of new features in the (now) `latest` which includes UI updates re design UI kit updates. It will also make some styles "opt-in" like bullet lists and certain typography text styles (Display, etc. 👈 About to add that now to the docs.).
